lib/rsvp/asap.js

```diff
@@ -16,7 +16,8 @@ export default function asap(callback, arg) {
 var browserWindow = (typeof window !== 'undefined') ? window : undefined;
 var browserGlobal = browserWindow || {};
 var BrowserMutationObserver = browserGlobal.MutationObserver || browserGlobal.WebKitMutationObserver;
-var isNode = typeof process !== 'undefined' && {}.toString.call(process) === '[object process]';
+var isNode = typeof window === 'undefined' &&
```

Comment:
as mentioned in ember, do these environments expose process in web workers / service workers?